### DISCUSSION TITLE
BitBucket cloud basic authentication is no longer supported

### DISCUSSION THREAD

**Discussion by tmontovaneli** [renovatebot/renovate#16288] [Suggest an Idea]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

The latest through docker

### Please select which platform you are using if self-hosting.

Bitbucket Cloud (bitbucket.org)

### If you're self-hosting Renovate, tell us what version of the platform you run.

Bitbucket Cloud

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

BB -> BitBucket

BB has announced that basic auth is no longer supported. [Reference](https://community.atlassian.com/t5/Bitbucket-articles/Announcement-Bitbucket-Cloud-account-password-usage-for-Git-over/ba-p/1948231)

I've tried to fix the issue, but I could not. I managed to get the token using the new endpoint, but I could not figure out how to pass it through downwards requests.

[BB reference for Auth 2.0](https://developer.atlassian.com/cloud/bitbucket/oauth-2/)

One can find what I've done [here](https://github.com/renovatebot/renovate/pull/16280/files)

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment]
Doesn't it work with an app password instead?

**Comment by viceice** [comment]
i don't see a bug here, should be converted to discussion

https://docs.renovatebot.com/modules/platform/bitbucket/#authentication

**Reply by rarkins** [reply]
Done. @tmontovaneli our instructions say to use an app password, not actual password.

**Comment by tmontovaneli** [comment]
Apologies, folks. My bad. You're right! It's working!
